core/src/commonMain/kotlin/dev/kryptonreborn/blockfrost/mempool/model/MempoolTransactionDetails.kt

```diff
@@ -0,0 +1,132 @@
+package dev.kryptonreborn.blockfrost.mempool.model
+
+import com.ionspin.kotlin.bignum.integer.BigInteger
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Transaction Details
+ *
+ * @property tx Transaction data
+ * @property inputs List of transaction inputs
+ * @property outputs List of transaction outputs
+ * @property redeemers List of redeemers
+ */
+@Serializable
+data class MempoolTransactionDetails(
+    val tx: TransactionData,
+    val inputs: List<TransactionInput>,
+    val outputs: List<TransactionOutput>,
+    val redeemers: List<Redeemer>,
+)
+
+/**
+ * Transaction Data
+ *
+ * @property hash Transaction hash
+ * @property outputAmount List of output amounts
+ * @property fees Fees of the transaction in Lovelaces
+ * @property deposit Deposit within the transaction in Lovelaces
+ * @property size Size of the transaction in Bytes
+ * @property invalidBefore Left (included) endpoint of the timelock validity intervals
+ * @property invalidHereafter Right (excluded) endpoint of the timelock validity intervals
+ * @property utxoCount Count of UTXOs within the transaction
+ * @property withdrawalCount Count of the withdrawals within the transaction
+ * @property mirCertCount Count of the MIR certificates within the transaction
+ * @property delegationCount Count of the delegations within the transaction
+ * @property stakeCertCount Count of the stake keys (de)registration within the transaction
+ * @property poolUpdateCount Count of the stake pool registration and update certificates within the transaction
+ * @property poolRetireCount Count of the stake pool retirement certificates within the transaction
+ * @property assetMintOrBurnCount Count of asset mints and burns within the transaction
+ * @property redeemerCount Count of redeemers within the transaction
+ * @property validContract True if contract script passed validation
+ */
+@Serializable
+data class TransactionData(
+    val hash: String,
+    @SerialName("output_amount") val outputAmount: List<OutputAmount>,
+    val fees: String,
+    val deposit: String,
+    val size: Int,
+    @SerialName("invalid_before") val invalidBefore: String?,
```

Comment:
long?

Reply:
invalid_before string nullable required
Left (included) endpoint of the timelock validity intervals

Reply:
nó là string mà